test/Microsoft.AspNetCore.Server.IISIntegration.Tests/IISMiddlewareTests.cs

```diff
@@ -79,14 +78,64 @@ public void UrlDelayRegistered()
                 .UseSetting("TOKEN", "TestToken")
                 .UseSetting("PORT", "12345")
                 .UseSetting("APPL_PATH", "/")
-                .UseIISIntegration();
+                .UseIISIntegration()
+                .Configure(app =>
+                {
+                    app.Run(context => Task.FromResult(0));
+                });
 
             Assert.Null(builder.GetSetting(WebHostDefaults.ServerUrlsKey));
 
             // Adds a server and calls Build()
             var server = new TestServer(builder);
 
-            Assert.Equal("http://localhost:12345/", builder.GetSetting(WebHostDefaults.ServerUrlsKey));
+            Assert.Equal("http://localhost:12345", builder.GetSetting(WebHostDefaults.ServerUrlsKey));
```

Comment:
Slight behaviour change. Paths that represent empty paths will no longer be appended to the url address that the server sees.